packages/material-ui-lab/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -184,6 +184,16 @@ export type AutocompleteClassKey =
   | 'groupLabel'
   | 'groupUl';
 
+/**
+ *
+ *
+ * Demos:
+ * - [Autocomplete](https://material-ui.com/components/autocomplete/)
```

Comment:
I believe that the markdown specification requires a blank line to format a list. At least, it has bee a hard requirement with Crowdin and is also how prettier force the format.

```suggestion
 * Demos:
 *
 * - [Autocomplete](https://material-ui.com/components/autocomplete/)
```

Reply:
Yep this is patched by vscode. A little bit confusing why you want conform markdown (that is patched) but not in i18n where it isn't. I can fix it for this case.

Reply:
The motivation would be about consitency. As we have no linting on the markdown of the documentation, the more we get used to this format, the less likely we will break the translations :)